NAMESPACE

```diff
@@ -9,6 +9,7 @@ export(as_listing)
 export(get_keycols)
 export(is_keycol)
 export(listing_dispcols)
+export(pag_tt_indices)
```

Comment:
the tt in this function name stands for `TableTree`, so this should not be named this in `rlistings`